.github/workflows/ci.yml

```diff
@@ -187,13 +187,16 @@ jobs:
           - name: PostgreSQL 11, PostGIS 3.1
             POSTGRES_IMAGE: postgis/postgis:11-3.1
             NODE_VERSION: 16.13.0
-          - name: PostgreSQL 12, PostGIS 3.1
+          - name: PostgreSQL 11, PostGIS 3.2
+            POSTGRES_IMAGE: postgis/postgis:11-3.2
+            NODE_VERSION: 16.13.0
+          - name: PostgreSQL 12, PostGIS 3.2
             POSTGRES_IMAGE: postgis/postgis:12-3.1
```

Comment:
Is this correct? It says "PostGIS 3.2", but the image is  "-3.1". Also in some lines below.

Reply:
It's not correct, will update